src/components/instantlaunches/launch/index.js

```diff
@@ -0,0 +1,93 @@
+ */
+import React, { useState } from "react";
+
+import { useQuery } from "react-query";
+
+import {
+    GET_INSTANT_LAUNCH_FULL_KEY,
+    getFullInstantLaunch,
+} from "serviceFacades/instantlaunches";
+import {
+    getResourceUsageSummary,
+    RESOURCE_USAGE_QUERY_KEY,
+} from "serviceFacades/dashboard";
+import { getUserQuota } from "common/resourceUsage";
+import { useConfig } from "contexts/config";
+import { useUserProfile } from "contexts/userProfile";
+import globalConstants from "../../../constants";
+import { useTranslation } from "i18n";
+import isQueryLoading from "components/utils/isQueryLoading";
+import InstantLaunchButtonWrapper from "components/instantlaunches/InstantLaunchButtonWrapper";
+import withErrorAnnouncer from "components/error/withErrorAnnouncer";
+import LoadingAnimation from "components/vice/loading/LoadingAnimation";
+
+import ErrorTypography from "components/error/ErrorTypography";
+import DEErrorDialog from "components/error/DEErrorDialog";
+
+const InstantLaunchStandalone = (props) => {
+    const { id: instant_launch_id, showErrorAnnouncer } = props;
+    const [config] = useConfig();
+    const [userProfile] = useUserProfile();
+    const [computeLimitExceeded, setComputeLimitExceeded] = useState(
+        !!config?.subscriptions?.enforce
+    );
+    const [errorDialogOpen, setErrorDialogOpen] = useState(false);
+    const { t } = useTranslation(["instantlaunches", "common"]);
+
+    const { data, status, error } = useQuery(
+        [GET_INSTANT_LAUNCH_FULL_KEY, instant_launch_id],
+        () => getFullInstantLaunch(instant_launch_id)
+    );
+
+    const { isFetching: isFetchingUsageSummary } = useQuery({
+        queryKey: [RESOURCE_USAGE_QUERY_KEY],
+        queryFn: getResourceUsageSummary,
+        enabled: !!config?.subscriptions?.enforce && !!userProfile?.id,
+        onSuccess: (respData) => {
+            const computeUsage = respData?.cpu_usage?.total || 0;
+            const subscription = respData?.subscription;
+            const computeQuota = getUserQuota(
+                globalConstants.CPU_HOURS_RESOURCE_NAME,
+                subscription
+            );
+            setComputeLimitExceeded(computeUsage >= computeQuota);
+        },
+        onError: (e) => {
+            showErrorAnnouncer(t("common:usageSummaryError"), e);
+        },
+    });
+
+    const isLoading = isQueryLoading([status, isFetchingUsageSummary]);
+
+    if (isLoading) {
+        return <LoadingAnimation />;
+    } else if (error) {
+        return (
+            <>
+                <ErrorTypography
+                    errorMessage={error.message}
+                    onDetailsClick={() => setErrorDialogOpen(true)}
+                />
+                <DEErrorDialog
+                    open={errorDialogOpen}
+                    errorObject={error}
+                    handleClose={() => setErrorDialogOpen(false)}
+                />
+            </>
```

Comment:
This looks basic enough that this could be replaced with an `ErrorTypographyWithDialog`. 

See `components/apps/savedLaunch/SavedLaunchListing` for an example (I guess `AppInfo` wasn't the best example for `ErrorTypography`, which seems like it could also use an `ErrorTypographyWithDialog` instead).